src/views/collection-view/components/collection-tutorial-list/collection-tutorial-list.module.css

```diff
@@ -0,0 +1,47 @@
+.listRoot {
+  --min-card-width: 270px;
+
+  background: var(--token-color-surface-faint);
+  box-shadow: var(--token-surface-base-box-shadow);
+  border-radius: 6px;
+  display: grid;
+  gap: 24px;
+
+  /**
+  * Automatically use more columns, as long as columns are >= min-column-width.
+  * Note: minimum size may be < min-column-width, if 100% < min-column-width.
+  * This ensures that on small viewports, content will never overflow.
+  */
+  grid-template-columns: repeat(
+    auto-fit,
+    minmax(min(100%, var(--min-card-width)), 1fr)
+  );
+  list-style: none;
+  margin: 0;
+  padding: 16px;
+
+  @media (--dev-dot-tablet-up) {
+    /**
+     * On larger viewports, where our content area reaches up to 780px width,
+     * cards are meant to be in two columns.
+     * 
+     * Note: we could explicitly set 2 columns here instead - it would achieve
```

Comment:
I don't know if this merits such a long comment, but felt it was worth explaining the rationale.

Reply:
I think this is great, thanks for explaining your rationale